.github/workflows/pullRequest.yml

```diff
@@ -10,3 +10,10 @@ jobs:
       - uses: stefanoeb/eslint-action@1.0.2
         with:
           files: services/
+
+  unitTest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: npm install
```

Comment:
you should use `npm ci` (clean install), it will install the exact dependencies that are in your package-lock.json

Reply:
Same goes for any installs done in actions, for both this and the FE project